rest/api.go

```diff
@@ -404,13 +404,15 @@ type DatabaseRoot struct {
 	State                         string   `json:"state"`
 	ServerUUID                    string   `json:"server_uuid,omitempty"`
 	RequireResync                 []string `json:"require_resync,omitempty"`
+	InitializationActive          bool     `json:"init_in_progress,omitempty"`
 }
 
 type DbSummary struct {
-	DBName string `json:"db_name"`
-	Bucket string `json:"bucket"`
-	State  string `json:"state"`
-	Reason string `json:"reason,omitempty"`
+	DBName               string `json:"db_name"`
+	Bucket               string `json:"bucket"`
+	State                string `json:"state"`
+	InitializationActive bool   `json:"init_in_progress,omitempty"`
+	Reason               string `json:"reason,omitempty"`
```

Comment:
Did you plan to change `Reason` to `RequireResync` in this PR? This was only in 3.1.7 and Isaac said it was fine to change this before rollout.